src/tools/compiletest/src/header.rs

```diff
@@ -1069,11 +1055,27 @@ fn ignore_llvm(config: &Config, line: &str) -> bool {
                 panic!("Malformed LLVM version range: max < min")
             }
             // Ignore if version lies inside of range.
-            actual_version >= v_min && actual_version <= v_max
+            return actual_version >= v_min && actual_version <= v_max;
         } else {
-            false
+            return false;
```

Comment:
Remove the `else` branch: if there's no `min-llvm-version` then we should continue on to the next check below.